migrations/20210305124354_create_initial_tables.down.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE companies;
+DROP TABLE jobs;
+DROP TABLE social_accounts;
+DROP TABLE users;
```

Comment:
👍

Reply:
have you successfully ran `up` and `down` migrations?

Reply:
Yes, but the migrations need some work.  The user id returned from Slack is a string and it sounded like you were thinking we didn't need a `users` table.  I didn't intend to include the migrations in this pull request but was working in a hurry and branched from the previous PR.